packages/lib/src/components/Dropin/components/DropinComponent.tsx

```diff
@@ -157,7 +188,29 @@ export class DropinComponent extends Component<DropinComponentProps, DropinCompo
                     <div className={`adyen-checkout__dropin adyen-checkout__dropin--${status.type}`}>
                         {isRedirecting && status.props.component && status.props.component.render()}
                         {isLoading && status.props && status.props.component && status.props.component.render()}
-                        {!!hasPaymentMethodsToBeDisplayed && (
+
+                        {showFastlane && (
+                            <Fragment>
+                                <PaymentMethodList
+                                    isLoading={isLoading}
+                                    paymentMethods={fastlanePaymentElement}
+                                    activePaymentMethod={activePaymentMethod}
+                                    cachedPaymentMethods={cachedPaymentMethods}
+                                    onSelect={this.handleOnSelectPaymentMethod}
+                                    openFirstPaymentMethod
+                                    showRadioButton={this.props.showRadioButton}
+                                />
+                                <Button
```

Comment:
Maybe hide this button when the there is no regular PMs to display - `!hasPaymentMethodsToBeDisplayed`?